### PR TITLE
fix(vars/longevityPipeline.groovy): Add unlink of ./latest just before test start

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/I5mjx0Al/1656-improve-identification-of-the-current-run-in-the-send-email-stage

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (CI)~~
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by dkropachev** [comment]
Test runs:
Fail before test - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dmitry-byo-longevity/285/
Fail during the test - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dmitry-byo-longevity/286/
Success run - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dmitry-byo-longevity/287/

File: vars/longevityPipeline.groovy [hunk 1]
```diff
@@ -12,6 +12,7 @@ def runSctTest(Map params){
     #!/bin/bash
     set -xe
     env
+    unlink ./latest 2>/dev/null
```
[the remark this answers is not in the document]
Comment (bentsi) [reply vars/longevityPipeline.groovy:15]: why not using `rm -fv` it doesn't exit with return code >0 if the file doesn't exist

**Comment by dkropachev** [comment]
> 1. need to add to all our pipelines that run tests: Byo, perf-regression, artifacts etc
> 2. why do we need `test-cases/longevity/longevity-sanity-check-15min.yaml`?

1. Added
2. To run quick sanity check test for with longevity pipeline.

**Comment by bentsi** [comment]
please remove `longevity-sanity-check-15min.yaml`, we create all the test jobs based on all the yamls that are located in the longevity dir + we have `test-cases/PR-provision-test.yaml` that is a "sanity" yaml

**Comment by dkropachev** [comment]
> please remove `longevity-sanity-check-15min.yaml`, we create all the test jobs based on all the yamls that are located in the longevity dir + we have `test-cases/PR-provision-test.yaml` that is a "sanity" yaml

Done.

File: vars/longevityPipeline.groovy (hunk 1 above)
[the remark this answers is not in the document]
Comment (dkropachev) [reply vars/longevityPipeline.groovy]: > why not using `rm -fv` it doesn't exit with return code >0 if the file doesn't exist

Done.

[the remark this answers is not in the document]
Comment (bentsi) [reply vars/longevityPipeline.groovy]: ???

**Comment by bentsi** [comment]
please address unresolved comment

File: vars/longevityPipeline.groovy (hunk 1 above)
[the remark this answers is not in the document]
Comment (dkropachev) [reply vars/longevityPipeline.groovy]: > ???
I've changed unlink to rm -fv, it matches better for this case

**Comment by dkropachev** [comment]
> please address unresolved comment

As I can see it is addressed.

**Comment by bentsi** [comment]
you need to explain in the body of the commit message why you did this

**Comment by dkropachev** [comment]
> you need to explain in the body of the commit message why you did this

Done.

**Pull request merged**